tools/audio/extraction/audio_extract.py

```diff
@@ -114,13 +115,34 @@ def collect_soundfonts(audiobank_seg : memoryview, extracted_dir : str, version_
 
     return soundfonts
 
-def extract_samplebank(extracted_dir : str, sample_banks : List[Union[AudioTableFile, int]], bank : AudioTableFile,
-                       write_xml : bool):
+def aifc_extract_one_sample(base_path : str, sample : AudioTableSample):
+    aifc_path = f"{base_path}/aifc/{sample.filename}"
```

Comment:
The .half.wav extension is for rebuilding, there needs to be some way to select the makefile rule to use for selecting the half-frame vadpcm codec. I don't want to over-specialize sampleconv by making it sensitive to these extensions, and I wouldn't want to add an extension for "full"-frame vadpcm since that should be understood as default.